frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -88,72 +152,338 @@ export const UninstallOperatorModal: React.FC<UninstallOperatorModalProps> = ({
               },
               {},
               deleteOptions,
-            ).catch(() => Promise.resolve()),
+            ),
           ]
         : []),
       ...(removePlugins
         ? [k8sPatch(ConsoleOperatorConfigModel, consoleOperatorConfig, [patch])]
         : []),
     ];
+    const operandDeletionPromises = deleteOperands
+      ? operands.map((operand: K8sResourceCommon) => {
+          const model = modelFor(referenceFor(operand));
+          return k8sKill(model, operand, {}, deleteOptions);
+        })
+      : [];
+    if (operandDeletionPromises.length > 0) {
+      setOperandsDeleteInProgress(true);
+      // eslint-disable-next-line promise/catch-or-return
+      settleAllPromises(operandDeletionPromises).then(
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        ([fulfilledValues, rejectedReasons, results]) => {
```

Comment:
I believe you can leave off the values you don't need, e.g.

```suggestion
        ([, , results]) => {
```